tests/fixture/tmpnet/node_process.go

```diff
@@ -112,7 +112,26 @@ func (p *NodeProcess) Start(w io.Writer) error {
 		return fmt.Errorf("failed to remove stale process context file: %w", err)
 	}
 
+	// All argument are provided in the flags file
 	cmd := exec.Command(p.node.RuntimeConfig.AvalancheGoPath, "--config-file", p.node.getFlagsPath()) // #nosec G204
+
+	// Ensure process is detached from the parent process so that an error in the parent will not affect the child
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		Setsid: true,
+	}
+
+	// Redirect stdout and stderr to ensure the subprocess is not writing to
+	// the parent's stdout and stderr. Since a node is intended to run
+	// headless and should be logging everything to disk internally, its
+	// direct output can be ignored by sending to /dev/null.
+	devNull, err := os.OpenFile(os.DevNull, os.O_WRONLY, 0)
+	if err != nil {
+		return fmt.Errorf("failed to open /dev/null: %w", err)
+	}
+	defer devNull.Close()
+	cmd.Stdout = devNull
+	cmd.Stderr = devNull
```

Comment:
> // Stdout and Stderr specify the process's standard output and error.
	//
	// If either is nil, Run connects the corresponding file descriptor
	// to the null device (os.DevNull).

Do we need to be setting these?

Reply:
They should be `nil` afaict:

> // Command returns the Cmd struct to execute the named program with
// the given arguments.
//
// It sets only the Path and Args in the returned structure.